compiler/aarch64/codegen/ARM64Ops.hpp

```diff
@@ -0,0 +1,4634 @@
+/*******************************************************************************
+ * Copyright (c) 2018, 2018 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at http://eclipse.org/legal/epl-2.0
+ * or the Apache License, Version 2.0 which accompanies this distribution
+ * and is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception [1] and GNU General Public
+ * License, version 2 with the OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#ifndef ARM64OPS_INCL
+#define ARM64OPS_INCL
+
+typedef enum  {
+  ARM64Op_bad,	// invalid operation
+  ARM64Op_cbz,	//
+  ARM64Op_cbnz,	//
+  ARM64Op_cbz,	//
+  ARM64Op_cbnz,	//
+  ARM64Op_tbz,	//
+  ARM64Op_tbnz,	//
+  ARARM64Op_cbz,	//
+  ARM64Op_cbnz,	//
+  ARM64Op_cbz,	//
+  AM64Op_b_cond,	//
+  ARM64Op_svc,	//
+  ARM64Op_hvc,	//
+  ARM64Op_smc,	//
+  ARM64Op_brk,	//
+  ARM64Op_hlt,	//
+  ARM64Op_dcps1,	//
+  ARM64Op_dcps2,	//
+  ARM64Op_dcps3,	//
+  ARM64Op_msr,	//  	00000000,	// bad
+  	34000000,	// cbz
```

Comment:
These enum values are leading with a number. Will this even compile?

Reply:
Fixed. There was a formatting issue.